lib/rubyhub/config/templates/.frontend_description.txt

```diff
@@ -0,0 +1,9 @@
+### Before submitting the merge request make sure the following are checked
```

Comment:
description will be equal for both frontend abd backend teams. itshould be the same file.